Source/WebCore/page/CaptionUserPreferencesMediaAF.cpp

```diff
@@ -461,6 +462,18 @@ String CaptionUserPreferencesMediaAF::captionsDefaultFontCSS() const
     if (!name)
         return emptyString();
 
+    if (fontNameIsSystemFont(name.get())) {
+        if (CFStringHasPrefix(CFSTR(".AppleSystemUIFontMonospaced"), name.get()))
+            name = CFSTR("system-ui-monospaced");
```

Comment:
This should be `ui-monospace` per https://www.w3.org/TR/css-fonts-4/#valdef-font-family-ui-monospace

and since that isn't guaranteed to exist on a system, I'd add a fallback to the list (either `system-ui` or `monospace`).